rpcs3/Emu/RSX/Overlays/HomeMenu/overlay_home_menu_main_menu.cpp

```diff
@@ -81,7 +81,7 @@ namespace rsx
 				});
 			}
 
-			std::unique_ptr<overlay_element> recording = std::make_unique<home_menu_entry>(get_localized_string(localized_string_id::HOME_MENU_RECORDING));
+			std::unique_ptr<overlay_element> recording = std::make_unique<home_menu_entry>(get_localized_string(g_user_asked_for_recording ? localized_string_id::HOME_MENU_STOP_RECORDING : localized_string_id::HOME_MENU_START_RECORDING));
```

Comment:
that's not how recording works

Reply:
I'm testing stuff, don't review just yet

Reply:
There's g_recording_mode.
If it is recording_mode::stopped, you can theoretically start a recording.
If it is recording_mode::rpcs3, you can theoretically stop a recording.
If it is recording_mode::cell, you theoretically shouldn't be allowed to either start or stop a recording.

Reply:
I dropped this commit due to a corner case with hotkeys